io.github.cxong.cdogs-sdl.json

```diff
@@ -0,0 +1,36 @@
+{
+  "app-id": "io.github.cxong.cdogs-sdl",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cdogs-sdl",
+  "rename-icon": "cdogs-sdl",
+  "rename-appdata": "cdogs-sdl",
+  "rename-desktop-file": "cdogs-sdl.desktop",
+  "rename-appdata-file": "cdogs-sdl.appdata.xml",
```

Comment:
New appdata files should use metainfo instead. freedesktop.org/software/appstream/docs/chap-Quickstart.html